certificate_agent.py

```diff
@@ -63,21 +59,11 @@ def main():
 
         if manager.get_length() == 0:
             log.debug("{0} has no jobs".format(str(manager)))
-            time.sleep(SLEEP_TIME)
+            time.sleep(settings.QUEUE_POLL_FREQUENCY)
             continue
         else:
             log.debug('queue length: {0}'.format(manager.get_length()))
 
-        xqueue_body = {}
```

Comment:
It's only possible to remove this because the exception handling does a `raise`-or-`continue` and therefore no code will use any of the removed variables.
It's a matter of style; I don't mind leaving that code to list the variables that will be used, but I don't mind removing it either.

Reply:
Yep, that's why.